src/components/Config.tsx

```diff
@@ -163,16 +164,22 @@ const YamlNodeWithDescription = ({
 
   return (
     <div
-      style={{ paddingLeft: `${root ? 0 : INDENT_SIZE}px` }}
       id={createUrlHash(parentKey, name)}
-      className={"config-tagged-for-algolia"}
+      className={clsx("config-tagged-for-algolia", !root && "config-node-indent-size")}
     >
-      <div className={`description_word_wrap`} style={{ marginBottom: showDescription ? 10 : 0 }}>
+      <div className={"description_word_wrap"}>
         <button
           onClick={() => {
             setShowDescription(!showDescription);
           }}
-          className={`config-node with-value${showDescription ? "-active" : ""} clean-btn button--link notranslate`}
+          className={clsx(
+            "config-node",
```

Comment:
any particular reason all these have been split? cant have have them all in one string and only hte active one in another? or at least group stuff that belongs together, like config-node with-value and clean-btn button--link, no need to have this use to many lines

Reply:
no specific reason; I've merged all the individual class strings into one